backend/de.metas.business/src/main/java-legacy/org/compiere/model/MInvoice.java

```diff
@@ -1503,6 +1508,7 @@ public boolean reverseCorrectIt()
 		setDocAction(DOCACTION_None);
 		setC_Payment_ID(0);
 		setIsPaid(true);
+		setIsPartiallyPaid(false);
```

Comment:
@metas-ts  @dm-ps don't we need to set the openamt=0?

Reply:
it will be set from `de.metas.invoice.service.impl.AbstractInvoiceBL#testAllocation`